### ISSUE TITLE
Improving the UX of the "Organize Categories" window — drag'n'drop

### ISSUE THREAD

**Issue by Korb** [moneymanagerex/moneymanagerex#4549]
Add the ability to drag categories from one root category to another.

**Comment by vomikan** [comment]
Probably, this can be implemented. But it seems to me that there may be more harm than good.

**Comment by tactilis** [comment]
@vomikan
> Probably, this can be implemented. But it seems to me that there may be more harm than good.

Agreed.
 
 
@Korb
Judging by your issue https://github.com/moneymanagerex/moneymanagerex/issues/4552, I think you may be a fairly new user of MMEX.

Can I respectfully suggest that, rather than create a lot of new issues straight away, you use MMEX for a while to manage your accounts - and then, having got more familiar with all of the application's functionality, create issues for anything that really would be an enhancement.

There are sometimes good reasons why the application works in a particular way.

**Comment by Korb** [comment]
@tactilis, I will heed your advice.

However, many times I have come across a situation where getting used to some non-obvious things does not make it possible to look at the familiar with a fresh look, to critically evaluate what has become familiar. Especially if you are the father of this product. Parents prefer not to notice the shortcomings in their children. And that's fine.

**Comment by tactilis** [comment]
All noted @Korb. And I do agree to an extent.

> does not make it possible to look at the familiar with a fresh look, to critically evaluate what has become familiar. Especially if you are the father of this product.

Luckily, I am not.

If you were to look back over github issues you will find that I have uncovered a number of bugs and have reported several UX/UI issues - but only after having used MMEX to enter and analyse a years worth of transactions across multiple accounts.

Your apparently very simple request to add some hotkeys to the Organise Categories dialog has been implemented - but it has created some bugs (that I have yet to report) that were not in MMEX before. It's often that case that adding features is not as simple as it first seems.
